src/main/java/de/tum/in/www1/artemis/web/rest/admin/ManagementResource.java

```diff
@@ -1,39 +1,35 @@
-package de.tum.in.www1.artemis.web.rest;
+package de.tum.in.www1.artemis.web.rest.admin;
 
 import java.util.List;
 import java.util.Map;
 
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.security.access.prepost.PreAuthorize;
 import org.springframework.web.bind.annotation.*;
 
+import de.tum.in.www1.artemis.security.annotations.EnforceAdmin;
 import de.tum.in.www1.artemis.service.feature.Feature;
 import de.tum.in.www1.artemis.service.feature.FeatureToggleService;
 
 @RestController
-@RequestMapping(ManagementResource.ROOT_MANAGEMENT)
+@RequestMapping("api/admin/")
 public class ManagementResource {
```

Comment:
❓ Question: Is there are reason why this class has no Admin Prefix?

Reply:
Yes, see JavaDoc in @EnforceAdmin